sdk/metric/sdk.go

```diff
@@ -652,11 +460,16 @@ func (m *SDK) RecordBatch(ctx context.Context, kvs []core.KeyValue, measurements
 	// called.  Subsequent calls to acquireHandle will re-use the
 	// previously computed value instead of recomputing the
 	// ordered labels.
-	var labels labels
+	var labels label.Set
 	for i, meas := range measurements {
 		s := meas.SyncImpl().(*syncInstrument)
 
-		h := s.acquireHandle(kvs, &labels)
+		var labelsPtr *label.Set
+		if i > 0 {
+			labelsPtr = &labels
+		}
```

Comment:
This was revised. Note that the former code organization would not have cached label encodings for a batch because it was copying the Set struct. (Lint found this w "copylocks" analysis.)

Therefore, the SDK `record` now contains both a `Label` for the common case and a `*Label` for the batch case. PTAL.